src/commissaire_http/handlers/hosts.py

```diff
@@ -187,6 +189,12 @@ def create_host(message, bus):
         # pass this off to the investigator
         bus.notify('jobs.investigate', params={'address': address})
 
+        # Push the host to the Watcher queue
+        watcher_record = models.WatcherRecord(
+            address=address,
+            last_check=_dt.utcnow().isoformat())
+        bus.producer.publish(watcher_record.to_json(), 'jobs.watcher')
```

Comment:
> Note the use of publish. This is because the watcher is working like a traditional global FIFO queue and not an RPC mechanism.

Cool, that makes sense here.